src/Fixtures/en/theory.en.js

```diff
@@ -211,6 +211,15 @@ export const tactics = [
         illustration: 'https://zupimages.net/up/20/30/gbjz.jpg',
         time: '15:42',
       },
+      {
+        id: 2,
+        title: 'Decision Making',
+        text: 'Disc In',
+        video: '514895844',
```

Comment:
Je pense que le titre de la vidéo au début est un peu chelou

"Make a offense" ça ne se dit pas trop

"How to attack like a pro" plutôt ?